CONTRIBUTING.md

```diff
@@ -8,6 +8,30 @@ The fastest way to get started is by running the Tembo docker image, where PGMQ
 docker run -d --name postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 quay.io/tembo/pgmq-pg:latest
 ```
 
+## PGXN Installation
+
+PGXN is a centralized network to make it easier to install and manage PostgreSQL extensions.
+
+Install PGXN Client:
+
+You can install the PGXN client by following the instructions provided in the [PGXN Client Installation Guide](https://pgxn.github.io/pgxnclient/install.html)
+
+Install PGMQ using PGXN
+
+```bash
+pgxn install pgmq
+```
+
+Alternatively, you can manually install PGMQ by downloading and building from the source using the following commands:
+
+```
+curl -LO https://api.pgxn.org/dist/pgmq/1.4.2/pgmq-1.4.2.zip
+unzip pgmq-1.4.2.zip
+cd pgmq-1.4.2
+make
+sudo make install
```

Comment:
I would omit `sudo` here, and suggest instead that you add a comment up top that these commands should be run as the user who owns the Postgres install (e.g., often `root` on *nix systems), or use `sudo`.

Reply:
Done!